BaseFiles/Common/Common.csproj

```diff
@@ -4523,13 +4526,5 @@
     <Compile Include="dummy.cs" />
   </ItemGroup>
   <ItemGroup />
-  <ItemGroup>
-    <Folder Include="html\pages\control\widgets\homegenie\generic\locales\" />
-    <Folder Include="html\js\i18n\" />
-    <Folder Include="html\pages\configure\widgeteditor\" />
-    <Folder Include="html\js\codemirror-4.03\mode\css\" />
-    <Folder Include="html\js\codemirror-4.03\mode\htmlembedded\" />

```

Comment:
many other files have the same issue, deleted lines sometimes inserted back.
I think it's best to checkout latest HomeGenie code, add your files and re-applying modifications to the html interface page.